### ISSUE TITLE
Deploy contracts to ropsten

### ISSUE THREAD

**Issue by palango** [raiden-network/raiden#1731]
Currently raiden doesn't start because the new contracts aren't deployed yet.

- [ ] Deploy now contracts
  - ~~Blocked by raiden-network/raiden-contracts#141~~
- [ ] Verify new contracts
  - [ ] Update docs on contract verification: https://raiden-network.readthedocs.io/en/stable/smart_contracts.html#verifying-on-etherscan
- [ ] Update contract addresses in `raiden/constants.py` and `README.md`

**Comment by LefterisJP** [comment]
This should be finished by ulo's latest work.

**Comment by palango** [comment]
We still need to update the verification docs

**Comment by LefterisJP** [comment]
Sure go ahead. But I would have made this in a separate issue, as verifying is something for us, not for others (since others already see the verified contracts thanks to etherscan)

**Comment by palango** [comment]
Yeah, we can add this to the docs issue.
